src/components/search/results.tsx

```diff
@@ -1,49 +1,72 @@
-import React from 'react';
+import React, { memo, useEffect, useState } from 'react';
 import Image from 'next/image';
+import { Button, ESystemIconNames, TypographyScale } from '@mdb/flora';
 
 import { ResultsProps } from './types';
-import { dataStyles } from './styles';
+import { dataStyles, resultsStyles } from './styles';
 
 import Card, { getCardProps } from '../card';
 
 import { getURLPath } from '../../utils/format-url-path';
+import { DEFAULT_PAGE_SIZE } from './utils';
+import { hasEmptyFilterAndQuery } from '../../hooks/search/utils';
+
+const DefaultNoResultsFooter = <TypographyScale>No Results</TypographyScale>;
 
 // TODO: This isn't being memoized correctly.
-const Results: React.FunctionComponent<ResultsProps> = React.memo(
-    ({ data, isLoading, hasError, layout = 'list' }) => {
+const SearchResults: React.FunctionComponent<ResultsProps> = memo(
+    ({
+        results = [],
+        isValidating,
+        error,
+        searchString,
+        filters,
+        slug,
+        pageNumber,
+        updatePageMeta = () => {},
+        noResultsFooter = DefaultNoResultsFooter,
+        layout = 'list',
+        extraStyles = {},
+    }) => {
+        const maxPage = Math.ceil(
+            Math.max(results.length, 1) / DEFAULT_PAGE_SIZE
+        );
+        const [currentPage, setCurrentPage] = useState(
```

Comment:
Previously the pagination related stuff was in `useSearch` but, in my opinion, it makes a bit more sense for `useSearch` to only be concerned with everything leading up to the search & fetching the data. Pagination, on the other hand, isn't a parameter for executing the search and I think only the component responsible for displaying the results should be concerned with it.

Reply:
ah okay I wasn't aware of this, good catch! I'll add a comment

Reply:
Nice catch @mcolella14, thanks for explaining here. @max-melo While the new backend can use pagination, it isn't required. If you do not pass a page number, the search will return all results still -- so as Matt mentioned, only a comment is needed.